workshop/exercise-1/README.md

```diff
@@ -6,25 +6,20 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
 
 1. Install the IBM Cloud [command line interface](https://console.bluemix.net/docs/cli/reference/bluemix_cli/get_started.html#getting-started).
 
-2.  Log in to the IBM Cloud CLI. If you have a federated account, include the `[--sso]` flag.   
-    
+2.  Log in to the IBM Cloud CLI. If you have a federated account, include the `[--sso]` flag.
+
     ```bash
     ibmcloud login [--sso]
     ```
 
-    or if you have an api key, use:
-    ```bash
-    ibmcloud login --apikey {apikey}
-    ```
-    
 3.  Install the IBM Cloud Kubernetes Service plug-in.
     ```bash
     ibmcloud plugin install container-service -r Bluemix
     ```
 
 4. To verify that the plug-in is installed properly, run `ibmcloud plugin list`. The Container Service plug-in is displayed in the results as `container-service`.
 
-5.  Initialize the Container Service plug-in and point the endpoint to your region. For example when prompted, enter `5` for `us-east`.   
+5.  Initialize the Container Service plug-in and point the endpoint to your region. For this workshop, enter `6` for `us-south`. We will be using `us-south` for this workshop.
```

Comment:
We do normally use us-east.  Wonder if we can make this configurable at a high level somehow?

Reply:
We'd need something like a Makefile that takes this repo(a generic tutorial) and does site/date specific configuration and publishes a version of it for direct consumption. This is useful and I support it, but wasn't in scope for this PR.

This change is because when we set up users with trial accounts and promo codes, they can only provision free clusters in us-south, so I thought this would help avoid issues. We help people get unstuck at this stage every time we run the workshop.

Reply:
we always used us-east when we ran the workshop in the past which is why I don't want to simply change to us-south.   For istio workshop, we don't recommend to use free clusters as they are quite small and doesn't have support for ALB.  I would prefer to ask user to select one vs hard code it.